README.md

```diff
@@ -1,107 +1,163 @@
+    compile(name: 'IDmeWebVerify', ext: 'aar')
 }
 ```
 
-### Sample Project
-To run the sample project you'll need to replace the `clientID` and `redirectUri` values in `MainActivity.java` with those values that are stored withing your registered app at [http://developer.id.me](http://developer.id.me).
-
-## Execution
-Both the OAuth flow and verification occur through a WebView that initialized from withn the SDK. Upon successful completion the activity will close and the JSON string will be sent back to the calling activity.
-
-To start the `WebViewActivity` you must initialize a `IDmeWebVerify` object,
+## Setup
+### Step 1
+#### Using gradle
+You have to declare you redirect url in your application `build.gradle` file.
+In order to do that, you have to add the following code replacing `CUSTOM_SCHEME` with the redirect custom scheme inside `android.defaultConfig` block
+
+
+```groovy
+android {
+    // ...
+    defaultConfig {
+        // ...,
+        manifestPlaceholders = [
+            idmeAuthRedirectScheme: "CUSTOM_SCHEME"
+        ]
+    }
+    // ...
+}
+```
 
-    IDmeWebVerify webVerify = new IDmeWebVerify(clientID, redirectURI, affiliationType, activity);
-or
-    IDmeWebVerify webVerify = new IDmeWebVerify(clientID, redirectURI, affiliationType, activity, returnProperties);
+<!--
+#### Using manual way
+- TODO
+-->
 
-and call the `StartWebView()`method
+### Step 2
+You must call `IDmeWebVerify.initialize(Context appContext, String clientId, String clientSecret, String redirectUri)` before using the SDK. 
+This method should be called only once. You should do it in your Application initialization method (`onCreate` method) or if you haven't got defined an Application class, you can invoke it in your `MainActivity` initialization method.
 
-    webVerify.StartWebView();
+The params in the initializer are as follows:
+- `appContext`: The application context.
+- `clientId`: The clientId provided by I.me when registering the app at [http://developer.id.me](http://developer.id.me).
+- `clientSecret`: The clientSecret provided by ID.me when registering the app at [http://developer.id.me](http://developer.id.me).
+- `redirectUri`: The redirectUri provided to ID.me when registering your app at [http://developer.id.me](http://developer.id.me).
 
-The params in the constructor are as follows:
 
-- `clientID`: The clientID provided by ID.me when registering the app at [http://developer.id.me](http://developer.id.me).
-- `redirectURI`: The redirectURI provided to ID.me when registering your app at [http://developer.id.me](http://developer.id.me)
-- `affiliationType`: The type of group verfication that should be presented.
-- `activity`: The activity that is creating the Class Object.
-- `returnProperties`: Indicates returning the user properties or an access token which you can use to get user properties.
+## Execution
+Both the OAuth flow and verification occur through a device browser that initialized from within the SDK.
```

Comment:
I think it should say *(...) through a device browser that **is** initialized (...)*